### PR TITLE
Fix gradle plugin for correct generation federated schema

### PR DESCRIPTION
**Pull request by RoMiRoSSaN**
The current plugin does not fully fulfill its role. This is a small fix for him. If you have the opportunity to check the correctness of its operation, I would be grateful.
Unfortunately, this is the only option that worked as it should.

### CODE REVIEW

**Comment by jmartisk** [comment]
Thanks @RoMiRoSSaN , from a quick glance it makes sense, I'll give it some more proper testing as soon as I get some time

File: docs/gradle-plugin.md [hunk 1]
```diff
@@ -86,3 +86,7 @@ Configuration options
 
 -   `includeIntrospectionTypes` - Include the introspection types in the
     schema. Default false.
+
+-   `federationEnabled` - Enable GraphQL Federation. This is automatically
+    enabled if any Federation-related annotations are found in your application, otherwise the default is false.
```
Comment (jmartisk) [inline docs/gradle-plugin.md:91]: The gradle plugin doesn't automatically check for these, does it? (I don't see that anywhere)

Comment (RoMiRoSSaN) [reply docs/gradle-plugin.md:91]: I followed the example of maven plugin. Let me try to do automatic detection of federation annotations.

**Comment by RoMiRoSSaN** [comment]
@jmartisk I added autodetect of federation directives. Please, check it

**Review by jmartisk — APPROVED** [review]
LGTM, thanks :)

**Pull request merged**